doc/_static/css/custom.css

```diff
@@ -7,15 +7,64 @@ body {
     .toctree-wrapper > ul {
         columns: 2;
     }
+    .toc-outside-links {
+        columns: 2;
+    }
+}
+.item-framing-box {
+    align-items: flex-start;
+    display: flex;
+    margin: 10px;
+}
+.item-framing-box .single-col-box {
```

Comment:
I'm not yet sure on this approach. My first reaction is to use child rules rather than applying to all descendants, but I don't yet fully understand your intent.

Reply:
You're right, nice tip. I swapped them to child selectors on seeing that child selectors are more efficient than descendant selectors and less likely to cause someone else issues down the line.